app/views/posts/index.html.erb

```diff
@@ -1,10 +1,18 @@
-
 <!DOCTYPE html>
 <html>
 <head>
-  <title>My Blog</title>
+  <title>Blog Posts</title>
 </head>
 <body>
-  <h1>Welcome to My blog</h1>
+  <h1><%= @user.name %>'s Posts</h1>
+
+  <% @posts.each do |post| %>
+    <div>
+      <h3><%= link_to post.title, post_path(post) %></h3>
+      <p><%= post.text %></p>
+    </div>
+  <% end %>
+  
+  <li><a href="/">back to home</a></li>
```

Comment:
- [ ] It is nice to keep to design guidelines, kindly match your UI to the given wireframe
in your design :point_down: 
![Screenshot from 2023-11-25 11-50-08](https://github.com/paulshaun-kdtk/blog-posts/assets/66097371/92d49442-def4-4ee5-8752-5847f03a6724)
given wireframe :point_down: 
![Screenshot from 2023-05-11 17-37-59](https://github.com/paulshaun-kdtk/blog-posts/assets/66097371/ed9c0a9e-869c-43ad-8752-9469923ae441)